Platform/CoffeelakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1752,6 +1752,7 @@ SaveNvsData (
   // if they match, no need to update again.
   //
   if (CompareMem ((VOID *)(UINTN)Address, Buffer, Length) == 0) {
+    DEBUG ((DEBUG_INFO, "MRC data matches in flash, skip update!\n"));

```

Comment:
I would suggest to return different status code.  core code will print the status.
In this way, it saves size.

Reply:
No, we don't recreate new status code. Instead, try to reuse existing ones.  We actually tried to use ALREADY_STARTED status to indicate that. So if you see ALREADY_STARTED print status, you know the MRC NVS matches and saving is skipped.

Reply:
yes, I can see "Already started" which is getting printed in core code for MRC fast boot.  thanks for suggestion
 
    First flash boot:
       MRC data successfully cached to 0x1FF000
       Save MRC Training Data (0x8D131480 0x0018E0) ...Success

    MRC fast boot:
       Save MRC Training Data (0x8D131398 0x0018E0) ...Already started